src/lib/plugins/header.js

```diff
@@ -5,5 +5,7 @@ module.exports.player = function (player, serv, settings) {
       footer: JSON.stringify(footer)
     })
 
-  player.playerlistUpdateText(settings['player-list-text'].header, settings['player-list-text'].footer)
+  if (settings['player-list-text']) {
```

Comment:
Can you make it use good defaults instead of not calling the method at all ?

Reply:
Ok sure

Reply:
vanilla does not always use a custom player list so in my opinion its better that way

Reply:
Ah yeah then fine let's keep it that way

Reply:
Okay, reverted it.